markdown/bi-tools/DbVisualizer.md

```diff
@@ -0,0 +1,45 @@
+### DbVisualizer
+
+Before proceeding, ensure you have [configured your environment](../setup/configuration.md).
+
+Download the latest version of [DbVisualizer](https://www.dbvis.com/) to use the Driver in DbVisualizer.
+
+**For Mac you must launch your BI tool through terminal to have the environment variables loaded.**
```

Comment:
Description how to launch and specify variables (as in above)

Reply:
Same comment, do we care about command line anymore? Could we just remove these statements entirely now?